src/Entity/OperationRepository.php

```diff
@@ -48,6 +48,26 @@ public function save($operation)
     }
 
 
+    public function findVendorOperationsByPaymentVoucherId(OperationInterface $operation){
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this open curly brace to the beginning of the next line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1808)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Put one space between the closing parenthesis and the opening curly brace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1808)